sources/bing_webmaster/README.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Bing Webmaster
+description: dlt source for Bing Webmaster API
+keywords: [bing, bing webmasters, bing webmaster tools]
+---
+
+
+# Bing Webmaster
+
+This source allows site owners to retrieve the organic traffic to their pages via Bing.
+[Bing Webmaster](https://www.bing.com/webmasters/tools/) is a free service as part of Microsoft's Bing search engine. It allows webmasters to add their websites to the Bing index crawler, see their site's performance in Bing searches (including chat). This source reports the clicks, impressions and their respective average ranks.
+
+Resources that can be loaded using this verified source are:
+
+| Name             | Description                                                                     |
+| ---------------- | --------------------------------------------------------------------------------|
+| page_stats       | retrieves weekly traffic statistics for top pages belonging to a site_url       |
+| page_query_stats | retrieves weekly trafic statistics per query for each pair of page and site_url |
+
+
+## Initialize the pipeline
+
+```bash
+dlt init bing_webmaster duckdb
+```
+
+Here, we chose duckdb as the destination. Alternatively, you can also choose redshift, bigquery, or
+any of the other [destinations](https://dlthub.com/docs/dlt-ecosystem/destinations/).
+
+## Add credentials
```

Comment:
Can we also add a notion of secret name which should be in secrets.toml?

Reply:
Good point, I just did that. Running the latest commit eeb1a9b would require that you change in your secrets.toml from `bing_webmaster_api_key` to `api_key`.